fcp-defra-id-stub/users.json

```
@@ -376,6 +376,18 @@
           "name": "Test Farm"
         }
       ]
+    },
+    {
+      "crn": 1062311181,
+      "firstName": "James",
+      "lastName": "Test-Farmer",
+      "organisations": [
+        {
+          "organisationId": "31",
```

Should make this a different id, e.g.

```suggestion
          "organisationId": "32",
```